Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM photon:2.0 as builder
-MAINTAINER Venil Noronha <veniln@vmware.com>
-
-RUN tdnf install -y openssl-1.0.2o-3.ph2.x86_64
-RUN tdnf install -y go-1.10.3-1.ph2.x86_64
+RUN tdnf --refresh install -qy wget tar gzip git && \
+  wget -P /tmp -q https://dl.google.com/go/go1.12.1.linux-amd64.tar.gz && \
+  tar -C /usr/local -xzf /tmp/go1.12.1.linux-amd64.tar.gz && \
+  mkdir -p /root/go/src /root/go/bin
+ENV GOPATH=/root/go
+ENV GOBIN=/root/go/bin
+ENV GO111MODULE=on
 WORKDIR /root/go/src/github.com/vmware/wavefront-adapter-for-istio/
 COPY ./ .
-RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -v -o bin/wavefront ./wavefront/cmd/
+RUN CGO_ENABLED=0 GOOS=linux /usr/local/go/bin/go build -a -installsuffix cgo -v -o bin/wavefront ./wavefront/cmd/ >/dev/null 2>&1
```

Comment:
What's the reason for log redirection?

Reply:
Since we deleted the `vendor` directory, the `go build` procedure downloads a ton of dependencies and every one of it is logged. I wasn't sure we wanted to see all of that. Should I enable it? I'm not sure we'll find any wonderful info in there.

Reply:
I'd still retain it as it might contain useful information to debug on the CI system.